### PR TITLE
adding typings for react-measure

### PR DESCRIPTION
**Pull request by chrisdrackett**

### CODE REVIEW

**Comment by TrySound** [comment]
@chrisdrackett react-measure provides types in release candidate.

**Comment by chrisdrackett** [comment]
@TrySound thats true, but the RC is a full rewrite, I would assume it will have a different version number and api.

File: definitions/npm/react-measure_v2.x.x/flow_v0.53.x-/react-measure_v2.x.x.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * This is an autogenerated libdef stub for:
+ *
+ *   'react-measure'
+ *
+ * Fill this stub out by replacing all the `any` types.
+ *
+ * Once filled out, we encourage you to share your work with the
+ * community by sending a pull request to:
+ * https://github.com/flowtype/flow-typed
+ */
```
Comment (AndrewSouthpaw) [inline definitions/npm/react-measure_v2.x.x/flow_v0.53.x-/react-measure_v2.x.x.js]: Delete this generated comments.

File: definitions/npm/react-measure_v2.x.x/test_react-measure.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// @flow
```
Comment (AndrewSouthpaw) [inline definitions/npm/react-measure_v2.x.x/test_react-measure.js]: Put test file under `flow_v0.53.x-` as well.

File: definitions/npm/react-measure_v2.x.x/test_react-measure.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+import * as React from 'React'
+import Measure from 'react-measure'
+
+const fail = () => (
+  // $ExpectError
+  <Measure bounds={'string'} />
+)
```
Comment (AndrewSouthpaw) [inline definitions/npm/react-measure_v2.x.x/test_react-measure.js]: use `describe` and `it` as discussed in the contributing guide, please. :)

File: definitions/npm/react-measure_v2.x.x/test_react-measure.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+// @flow
+
+import * as React from 'React'
+import Measure from 'react-measure'
+
```
Comment (AndrewSouthpaw) [inline definitions/npm/react-measure_v2.x.x/test_react-measure.js]: Also test a success.

File: definitions/npm/react-measure_v2.x.x/flow_v0.53.x-/react-measure_v2.x.x.js [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+    innerRef?: (HTMLElement) => void,
+    onResize?: (ContentRect) => void,
+    children?: ({
+      measureRef: () => void,
+      measure: () => void,
+      contentRect: ContentRect,
+    }) => React$Node,
+  }> {}
+
+  declare module.exports: typeof Measure
```
Comment (AndrewSouthpaw) [inline definitions/npm/react-measure_v2.x.x/flow_v0.53.x-/react-measure_v2.x.x.js]: Don't mix CJS and ESM style exports: https://flow.org/en/docs/libdefs/creation/#toc-declaring-an-es-module

**Comment by GAntoine** [comment]
@chrisdrackett Are you still looking to get this merged in?

**Comment by chrisdrackett** [comment]
@GAntoine yes, I just haven't had a chance to get back and update based on the feedback!

**Pull request closed without merging**